server/index.js

```diff
@@ -9,68 +9,91 @@ app.use(parser.json());
+          },
+          media: {
+            video: data.rows[0].videos.slice(1, data.rows[0].videos.length - 1).split(',').map((item, index) => ({ video: item, thumbnail: data.rows[0].thumbnails.slice(1, data.rows[0].thumbnails.length - 1).split(',')[index] })),
+            images: data.rows[0].images.slice(1, data.rows[0].images.length - 1).split(','),
+          },
+        },
+      ];
+      res.status(200).json(JSONData);
     }
   };
   db.getGame(game, callback);
 });
 
-let id = 10000000;
-app.post('/media', (req, res) => {
-  id += 1;
-  const game = {
-    proxyId: id,
-    summary: {
-      title: 'title',
-      splash: 'splash',
-      description: 'description',
-      reviews: { general: 'general', total: 4 },
-      releaseDate: 'release date',
-      developer: 'developer',
-      publisher: 'publisher',
-      tags: ['tags'],
-      percentage: 50,
-    },
-    media: {
-      video: [{ video: 'video', thumbnail: 'thumbnail' }],
-      images: ['images'],
-    },
-  };
-  db.postGame(game, (err, result) => {
-    if (err) {
-      console.error(err);
-    } else {
-      res.send(result);
-    }
-  });
-});
+// let id = 10000000;
+// app.post('/media', (req, res) => {
+//   id += 1;
+//   const game = {
+//     proxyId: id,
+//     summary: {
+//       title: 'title',
+//       splash: 'splash',
+//       description: 'description',
+//       reviews: { general: 'general', total: 4 },
+//       releaseDate: 'release date',
+//       developer: 'developer',
+//       publisher: 'publisher',
+//       tags: ['tags'],
+//       percentage: 50,
+//     },
+//     media: {
+//       video: [{ video: 'video', thumbnail: 'thumbnail' }],
+//       images: ['images'],
+//     },
+//   };
+//   db.postGame(game, (err, result) => {
+//     if (err) {
+//       console.error(err);
+//     } else {
+//       res.send(result);
+//     }
+//   });
+// });
 
-app.put('/media', (req, res) => {
-  const game = req.query;
-  console.log(req.body);
-  db.putGame({ proxyId: game }, { summary: req.body.summary }, (err, result) => {
-    if (err) {
-      console.error(err);
-    } else {
-      res.send(result);
-    }
-  });
-});
+// app.put('/media', (req, res) => {
+//   const game = req.query;
+//   console.log(req.body);
+//   db.putGame({ proxyId: game }, { summary: req.body.summary }, (err, result) => {
+//     if (err) {
+//       console.error(err);
+//     } else {
+//       res.send(result);
+//     }
+//   });
+// });
 
-app.delete('/media', (req, res) => {
-  const game = req.query;
-  db.deleteGame({ proxyId: game }, (err, result) => {
-    if (err) {
-      console.error(err);
-    } else {
-      res.send(result);
-    }
-  });
-});
+// app.delete('/media', (req, res) => {
+//   const game = req.query;
+//   db.deleteGame({ proxyId: game }, (err, result) => {
+//     if (err) {
+//       console.error(err);
+//     } else {
+//       res.send(result);
+//     }
+//   });
+// });
```

Comment:
are you intentionally leaving these endpoints non-functional?

Reply:
for now, yeah. I just haven't implemented them yet. For my front end to work all I need is the GET endpoint.